homeassistant/components/suez_water/sensor.py

```diff
@@ -0,0 +1,238 @@
+"""Sensor for Suez Water Consumption data."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, VOLUME_LITERS
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+CONF_COUNTER_ID = 'counter_id'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=720)
+SCAN_INTERVAL = timedelta(minutes=720)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Required(CONF_COUNTER_ID): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the sensor platform."""
+    username = config.get(CONF_USERNAME)
+    password = config.get(CONF_PASSWORD)
+    counter_id = config.get(CONF_COUNTER_ID)
+    _LOGGER.debug(
+        "Username is %s and counter_id is %s.",
```

Comment:
Avoid putting the username and the counter ID in the log file.

Reply:
Fixed